packages/uploadthing/src/internal/validate-request-input.ts

```diff
@@ -291,7 +293,7 @@ export const parseAndValidateRequest = (opts: {
     // FIXME: This should probably provide the full context at once instead of
     // partially in the `runRequestHandlerAsync` and partially in here...
     // Ref: https://discord.com/channels/@me/1201977154577891369/1207441839972548669
-    const contextValue = yield* fetchContext;
+    const contextValue = yield* FetchContext;
```

Comment:
Is there any way to fix this fixme comment here? Would prefer if we didnt have to init them as undefined higher up

Reply:
One way to do, and how I would probably approach it, is to split it into two seperate tags. One for the fetch instance and one for the base headers.

Reply:
yea seems simple enough